client/src/partials/general_ledger/general_ledger.js

```diff
@@ -102,12 +103,21 @@ function GeneralLedgerController(GeneralLedger, Sorting, Grouping, Filtering, Co
       displayName : 'TABLE.COLUMNS.DATE',
       headerCellFilter: 'translate',
       cellFilter : 'date:"mediumDate"',
-      filter : { condition : filtering.byDate }
+      filter : { condition : filtering.byDate },
+      footerCellTemplate:'<i></i>'
     },
-    { field : 'description', displayName : 'TABLE.COLUMNS.DESCRIPTION', headerCellFilter: 'translate' },
+    { field : 'description', displayName : 'TABLE.COLUMNS.DESCRIPTION', headerCellFilter: 'translate', footerCellTemplate:'<i></i>' },
     { field : 'account_number', displayName : 'TABLE.COLUMNS.ACCOUNT', headerCellFilter: 'translate' },
-    { field : 'debit_equiv', displayName : 'TABLE.COLUMNS.DEBIT', headerCellFilter: 'translate', cellTemplate : '/partials/templates/grid/debit_equiv.cell.html' },
-    { field : 'credit_equiv', displayName : 'TABLE.COLUMNS.CREDIT', headerCellFilter: 'translate', cellTemplate : '/partials/templates/grid/credit_equiv.cell.html' },
+    { field : 'debit_equiv', displayName : 'TABLE.COLUMNS.DEBIT', headerCellFilter: 'translate',
+      cellTemplate : '/partials/templates/grid/debit_equiv.cell.html',
+      aggregationType : uiGridConstants.aggregationTypes.sum,
+      footerCellFilter : 'currency'
```

Comment:
Actually, I just discovered you can pass custom filter arguments to the cellFilter.  [See this plunkr](http://plnkr.co/edit/972s2MSbFDw2DtnMOmnY?p=preview).

So, you can do this:
```js
footerCellFilter: 'currency:grid.appScope.enterprise.currency_id'
```
and make sure that the enterprise is bound to the scope!